signer/src/transaction_coordinator.rs

```diff
@@ -213,7 +217,10 @@ where
     }
 
     #[tracing::instrument(skip(self))]
-    async fn process_new_blocks(&mut self) -> Result<(), Error> {
+    async fn process_new_blocks(
+        &mut self,
+        with_delay: tokio::time::Duration,
+    ) -> Result<(), Error> {
```

Comment:
If you add the delay to the config you can delete this change and access the configuration via the context within the `self` reference.

Reply:
Resolved.